### PR TITLE
PORTALS-2943: Sage branding update

### PR DESCRIPTION
**Pull request by jay-hodgson**
Switch from Lato to DM Sans

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Update CancerComplexity Sage logo:
Screen Shot 2024-02-15 at 3 24 41 PM

**Comment by jay-hodgson** [comment]
Also update the Chal
Screen Shot 2024-02-15 at 4 02 41 PM
lenge portal:

File: packages/synapse-react-client/src/style/base/_fonts.scss [hunk 1]
```diff
@@ -3,30 +3,6 @@
...
-  font-style: normal;
-  font-weight: 400;
-  src: local('Lato Regular'), local('Lato-Regular'),
-    url(https://fonts.gstatic.com/s/lato/v14/S6uyw4BMUTPHjx4wXiWtFCc.woff2)
-      format('woff2');
-  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
-    U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215,
-    U+FEFF, U+FFFD;
-}
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap');
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/style/base/_fonts.scss:8]: imported css contains font-face definitions for latin-ext and latin for italic and regular

File: packages/synapse-react-client/src/components/Authentication/LoginPage.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const Tagline: StyledComponent<TypographyProps> = styled(Typography, {
   label: 'Tagline',
 })(({ theme }) => ({
   marginBottom: theme.spacing(2),
-  font: '300 24px/34px Lato, sans-serif',
+  font: '300 24px/34px DM Sans, sans-serif',
```
Comment (nickgros) [inline packages/synapse-react-client/src/components/Authentication/LoginPage.tsx:36]:
```suggestion
  font: '300 24px/34px "DM Sans", sans-serif',
```

**Review by nickgros — APPROVED** [review]
Just saw one place where I think we need quotes, LGTM otherwise.

**Pull request merged**